packages/design-system/src/components/css-value-list-item.stories.tsx

```diff
@@ -0,0 +1,165 @@
+import { css, styled, theme } from "../stitches.config";
+import { CssValueListItem } from "./css-value-list-item";
+import { Label } from "./label";
+import { SmallToggleButton } from "./small-toggle-button";
+import {
+  EyeconOpenIcon,
+  EyeconClosedIcon,
+  SubtractIcon,
+} from "@webstudio-is/icons";
+import * as React from "react";
+import { SmallIconButton } from "./small-icon-button";
+import {
+  FloatingPanelPopover,
+  FloatingPanelPopoverContent,
+  FloatingPanelPopoverTrigger,
+} from "./floating-panel-popover";
+import { StorySection, StoryGrid } from "./storybook";
+
+export default {
+  component: CssValueListItem,
+  args: { hidden: false, labelColor: "default", focused: false },
+  argTypes: {
+    hidden: { control: "boolean" },
+    focused: { control: "boolean" },
+    labelColor: {
+      options: ["default", "preset", "local", "remote"],
+      type: "inline-radio",
+    },
+  },
+};
+
+const Thumbnail = styled("div", {
+  width: theme.spacing[10],
+  height: theme.spacing[10],
+  backgroundImage: "linear-gradient(yellow, red)",
+});
+
+const ListItem = (props: {
+  hidden: boolean;
+  labelColor: "default" | "preset" | "local" | "remote";
```

Comment:
this is also the same with style source UI, but we put so far style source UI into builder, probably need to move it over to design system as well cc @TrySound